coreneuron/nrnmpi/nrnmpiuse.h

```diff
@@ -35,6 +35,11 @@ THE POSSIBILITY OF SUCH DAMAGE.
   #define NRNMPI 1
 #endif
 
+/* define to 1 if want multisend spike exchange available */
+#ifndef NRN_MULTISEND
```

Comment:
We have command line option to enable/disable multisend. In which case we want to disable compilation of multisend? Should I add that as CMake option? (if require)